tests/test_cli.py

```diff
@@ -64,17 +61,23 @@ class CodeGeneratorTests(unittest.TestCase):
     SETTINGS = _get_all_settings(Mock(indent=4))
     maxDiff = None
 
+    SHOW_ERRORS = os.environ.get("SHOW_ERRORS", False)
+    KEEP_GENERATED = os.environ.get("KEEP_GENERATED", False)
+    UPDATE_EXPECTED = os.environ.get("UPDATE_EXPECTED", False)
+    LINT = os.environ.get("LINT", False)
```

Comment:
This should be True by default, especially until the current golint min_confidence 1.0 problems are fixed, otherwise newcomers are going to be surprised when CI fails their tests when they passed locally.

You can however have the CLI disable it by default - newcomers should be running `tox`.

Reply:
You can also *rename* EXPECTED_COMPILE_FAILURES in this PR to be about lint only, as we now have test_unsupported where we no longer have existing test code which doesnt compile, and new test cases can be partially supported by only checking in some tests/expected/ or by using test_unsupported.